getting_started/Data_Input_Perf.md

```diff
@@ -0,0 +1,24 @@
+## Performance of Data Input Pipeline
+Overview of supported data input pipelines: https://github.com/google/maxtext/blob/main/getting_started/Data_Input_Pipeline.md
+
```

Comment:
Can you also include some information on the test setup? Mention that the datasets are in a GCS bucket (within the same region)? And the hosts are TPU VMs?

Reply:
Added test methods in the next paragraph